Dockerfile

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile1
+ARG VERSION=20.04
+FROM ubuntu:$VERSION
+ENV USERNAME dev
+# Insert your generated `pymatgen` API key below in place of
+# `insert-api-key-here` before building
+ENV MP_API_KEY insert-api-key-here
```

Comment:
might consider having this as a build arg, just so folks aren't tempted to keep their api keys in plaintext (or accidentally commit them like I sometimes do).  Docker doesn't complain though at build-time without a supplied arg, so I could see it being better either way.